src/cheri-pte-ext.adoc

```diff
@@ -141,11 +141,14 @@ bit value of the capability read. This will introduce additional traps during re
 |===
 |PTE.CW |PTE.CRG |Store/AMO
 | 0     | 0      | Page fault if stored tag is set
-| 0     | 1      | Page fault if stored tag is set, or hardware CW and CRG update^1^
+| 0     | 1      | Page fault if stored tag is set, or hardware CW and CRG update^2^
 | 1     | X      | Normal operation
 |===
 
-^1^ The choice here follows the pattern of whether to implement the _Svade_ extension
+^1^ The choice here is whether to take data dependent exceptions on loads or atomic operations.
+ The implementation is free to define when (if ever) it checks the tag.
```

Comment:
```suggestion
 It is legal for the implementation to fault even if the tag is set since this behaviour is only an optimization for software.
 This means it is also legal to only check the tag in certain conditions and conservatively fault otherwise.
```

Maybe something slightly more wordy to make it explicit that all of `always check/never check/only check for atomics/etc.` is a valid implemenetation choice.